turbo.json

```diff
@@ -19,7 +19,23 @@
       ]
     },
     "test-unit": {
-      "dependsOn": ["$TEST_FILES", "build"],
+      "dependsOn": ["build"],
+      "outputs": []
+    },
+    "test-integration-dev": {
+      "dependsOn": ["build"],
+      "outputs": []
+    },
+    "test-integration-cli": {
+      "dependsOn": ["build"],
+      "outputs": []
+    },
+    "test-integration-once": {
+      "dependsOn": ["build"],
+      "outputs": []
+    },
+    "test": {
+      "dependsOn": ["build"],
```

Comment:
What’s the purpose of this? Does it run every test regardless if unit/integration/e2e?